File: code/cfg/Task2.cs
using System;
namespace cfg{
public class Task2 : Task {
public const int __TYPE_ID__ = 2;
public int GetTypeid() { return __TYPE_ID__; }
public readonly int y;
public readonly string name;
public Task2(CSVStream fs) : base(fs) {
this.y = fs.GetInt();
this.name = fs.GetString();
}
}
}

File: code/cfg/Task2.java
package cfg;
public class Task2 extends Task {
public final static int __TYPE_ID__ = 2;
public final int getTypeid() { return __TYPE_ID__; }
public final int y;
public final String name;
public Task2(CSVStream fs) {
super(fs);
this.y = fs.getInt();
this.name = fs.getString();
}
}